include/deal.II/base/tensor_function.h

```diff
@@ -121,6 +132,11 @@ template <int rank, int dim, typename Number = double>
 class ConstantTensorFunction : public TensorFunction<rank, dim, Number>
 {
 public:
+  /**
+   * The scalar-valued real type used for representing time.
+   */
+  using typename TensorFunction<rank, dim, Number>::time_type;
```

Comment:
This might work, but it's of the old-style `using` kind that just injects a name instead of the new-style `typedef` kind. I'd prefer the latter.